README.md

```diff
@@ -15,13 +15,42 @@ Minimal Mistakes is a flexible two-column Jekyll theme, perfect for building per
 
 **Note:** The theme uses the [jekyll-include-cache](https://github.com/benbalter/jekyll-include-cache) plugin which will need to be installed in your `Gemfile` and added to the `plugins` array of `_config.yml`. Otherwise you'll encounter `Unknown tag 'include_cached'` errors at build.
 
+## Table of Contents
```

Comment:
Uh, what's the point of this?

Reply:
It's a Table of Contents for the readme. It helps orient people quickly to the topics of interest, and easily jump around. Explore this readme live and interactively here: https://github.com/ElectricRCAircraftGuy/minimal-mistakes/tree/upstream-theme_fixes.

It looks like this:
![image](https://user-images.githubusercontent.com/6842199/104830104-530d7f00-5830-11eb-940c-24d491c13a94.png)

Notice there's an arrow to "click to expand" and view the whole Table of Contents. It's super useful I think. I really like it.